renovate.json

```diff
@@ -2,5 +2,11 @@
   "$schema": "https://docs.renovatebot.com/renovate-schema.json",
   "extends": [
     "github>platform-engineering-org/.github"
+  ],
+  "packageRules": [
+    {
+      "matchDatasources": ["pypi"],
```

Comment:
Hmmm... not sure if that is needed. What is needed is to pin the version of one dependency.